webpack/scenes/Subscriptions/components/SubscriptionsToolbar/SubscriptionsToolbar.js

```diff
@@ -34,11 +35,11 @@ const SubscriptionsToolbar = ({
         <FormGroup className="toolbar-pf-filter">
           <SearchBar
             data={{
+              ...getControllerSearchProps('katello_subscriptions'),
```

Comment:
Correct way to add bookmark here:
```
         <FormGroup className="toolbar-pf-filter">
           <SearchBar
             data={{
-              autocomplete: {
-                url: '/katello/api/v2/subscriptions/auto_complete_search',
-                apiParams: { ...autocompleteQueryParams },
-              },
-              bookmarks: {},
+              ...getControllerSearchProps('/katello/api/v2/subscriptions', 'searchBar-katello_subscriptions', true, autocompleteQueryParams),
+              controller: 'katello_subscriptions',
             }}
             onSearch={onSearch}
             onSearchChange={updateSearchQuery}

Reply:
Thank you very much for your input. Could I please just have some questions so I understand it better?
- Isn't the line `bookmarks: {},` unnecessary when we override it with `getControllerSearchProps` anyway?
- Overall, why is putting the `autocomplete` first the correct way if the result is the same?

Reply:
Thank you very much for your input. I'm sorry about my last response if you read it, I somehow haven't noticed the +/- at the beginning of your lines so I was confused. I thought that my solution is less complicated, but now I see that your way is already used somewhere else in the code and it makes sense.